scalafix/rules/src/main/scala/fix/Infertypes.scala

```diff
@@ -1,10 +1,15 @@
 package fix
 
```

Comment:
The name `Infertypes` is not appropriate anymore since the rule does much more than just inferring type.

I would rename it to something more generic like `MigrationRule` or `Migrat3Rule`

Reply:
ok